### PR TITLE
Update lint-staged supported extensions

### PR DESCRIPTION
**Pull request by rivajunior**
**Changes proposed:**

- [x] Add
- [ ] Delete
- [ ] Fix
- [ ] Prepare

Since lint-staged [v12.1.0](https://github.com/okonet/lint-staged/releases/tag/v12.1.0) it support `.mjs` for its configuration.

I've removed the postfix of the icon filename and icon title tag to be more embracing, because lint-staged also support `lint-staged.config` filename with different extensions allowed.

### CODE REVIEW

**Review by robertohuertasm — APPROVED** [review]
LGTM 🚀 Thanks for your contribution ❤️

**Comment by robertohuertasm** [comment]
It seems there's something wrong with the tests...

**Review by robertohuertasm — CHANGES_REQUESTED** [review]
Let's check why the tests are failing

**Comment by rivajunior** [comment]
The merging of the two lintstaged objects fixes the tests

```typescript
{
      icon: 'lintstaged',
      extensions: ['lintstagedrc'],
      filenamesGlob: ['.lintstagedrc', 'lint-staged.config'],
      extensionsGlob: ['js', 'json', 'yaml', 'yml', 'cjs', 'mjs'],
      filename: true,
      format: FileFormat.svg,
},
```

but I think it would leading to a wrong result match in vscode, because the filename 'lint-staged.config' does't support 'json', 'yaml' and 'yml' extensions. That's why I've created two separated configs.

What you guys (@robertohuertasm @imartingon ) think?

**Comment by robertohuertasm** [comment]
Let me look into this.

File: src/iconsManifest/supportedExtensions.ts [hunk 1]
```diff
@@ -2374,13 +2374,19 @@ export const extensions: IFileCollection = {
...
+      extensions: ['.lintstagedrc'],
       filenamesGlob: ['.lintstagedrc'],
-      extensionsGlob: ['js', 'json', 'yaml', 'yml'],
+      extensionsGlob: ['js', 'json', 'yaml', 'yml', 'cjs', 'mjs'],
       filename: true,
       format: FileFormat.svg,
     },
+    {
+      icon: 'lintstaged',
+      filenamesGlob: ['lint-staged.config'],
```
Comment (robertohuertasm) [inline src/iconsManifest/supportedExtensions.ts:2386]: add `extensions: []`, that should do the trick to pass the tests.

**Review by robertohuertasm — CHANGES_REQUESTED** [review]

**Comment by xiaoxiangmoe** [comment]
Hi, all.
I found linstaged source codes has a list for config file names.

https://github.com/okonet/lint-staged/blob/e7b6412fb128f314346e28329c17a676cf691135/lib/loadConfig.js#L13-L25

```js
/**
 * The list of files `lint-staged` will read configuration
 * from, in the declared order.
 */
const searchPlaces = [
  'package.json',
  '.lintstagedrc',
  '.lintstagedrc.json',
  '.lintstagedrc.yaml',
  '.lintstagedrc.yml',
  '.lintstagedrc.mjs',
  '.lintstagedrc.js',
  '.lintstagedrc.cjs',
  'lint-staged.config.mjs',
  'lint-staged.config.js',
  'lint-staged.config.cjs',
]
```

So I copy this and create a PR.

see https://github.com/vscode-icons/vscode-icons/pull/2889/files#diff-d2e737a639e4d3439dac2e1e9f2597e1dd55a743854cdf2ea939dd5131afbf6fL2378-L2381

**Pull request closed without merging**